java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -4053,4 +4054,26 @@ public void geopos_returns_success() {
         assertEquals(testResponse, response);
         assertEquals(value, payload);
     }
+
+    @SneakyThrows
+    @Test
+    public void getbit_returns_success() {
+        // setup
+        String key = "testKey";
+        Long bit = 1L;
+        CompletableFuture<Long> testResponse = new CompletableFuture<>();
+        testResponse.complete(bit);
+
+        // match on protobuf request
+        when(commandManager.<Long>submitNewCommand(eq(GetBit), eq(new String[] {key, "1"}), any()))
```

Comment:
don't use the same value throughout.  The offset and return should not be the same.